makeflow/src/dag_file.h

```diff
@@ -16,16 +16,32 @@ See the file COPYING for details.
  * unique node, if any, that produces the file.
  */
 
+typedef enum {
+	DAG_FILE_STATE_UNKNOWN = 0,
+	DAG_FILE_STATE_EXPECT = 1,
```

Comment:
You can do without the = x. It is easier to maintain given that the numbers are in sequence anyway.